.github/workflows/release.yaml

```diff
@@ -1,17 +1,30 @@
-name: release-dekstop-app
+name: release-desktop-app
+
 on:
   push:
-     tags:
-       - 'v*.*.*'
+    tags:
+      - "v*.*.*"
+
 jobs:
-  create-release-linux:
+  create-draft-release:
```

Comment:
it's better to make the downstream jobs explicitly depend on this one

Reply:
Its already dependent. see the needs: tag in below jobs